### ISSUE TITLE
Interested in creating a more comprehensive Tor control module?

### ISSUE THREAD

**Issue by mattcollier** [Mowje/node-ths#10]
You are likely aware of the [stem Python controller](https://stem.torproject.org/api.html) for Tor.  I wonder if you'd be interested in working on a stem like nodejs module?

I've begun work on the control port class here: https://github.com/mattcollier/tor-control

**Comment by abenmrad** [comment]
Yeah I've heard of stem. Your idea is interesting and I guess worth working on. So you're planning on creating a new nodejs module for that purpose, correct?

Unfortunately I'm not sure I can spend time working on it, as I'm very busy over the next months. I've created node-ths out of necessity, just like I guess you plan on building tor-control out of necessity too. I guess I can however have a look on it from time to time and give feedback if you'd like.
